test/test-bootsample-non-parametric.jl

```diff
@@ -227,6 +226,14 @@ using StatsBase
 
     end
 
+    @testset "ConfInt" begin
+        @testset "All equal" begin
+            r = rand(10)
+            # In this case our count will always be the same because all values are positive
+            bs = bootstrap(x -> count(x .> 0.0), r, BasicSampling(n))
+            test_confint(bs)
```

Comment:
Maybe we can have a slightly stricter test here: `test_confint` only checks if `lower <= t0 <= upper`. In this case, we could explicitly check that `lower == t0 == upper` (e.g. by creating a new `test_confint_width0` function based on `test_confint` and adapting the test condition.